src/gsi/gesinfo.F90

```diff
@@ -255,6 +255,9 @@ subroutine gesinfo
            jcap=gfshead%jcap, ntrac=gfshead%ntrac, idvc=gfshead%idvc, &
            idsl=gfshead%idsl,   ncldt=gfshead%ncldt, iret=iret2)
 
+        ! FV3GFS write component does not include JCAP, infer from DIMY-2
```

Comment:
Why has this been commented out? I thought you needed to compute JCAP from DIMY-2?

Reply:
I compute JCAP in the get_gefs_for_regional.f90. 

! FV3GFS write component does not include JCAP, infer from DIMY-2
        if (njcap<0) njcap=latb-2

Reply:
Ok but does commenting this out interfere with the other (global) applications? This routine is not just for regional is it?